cli/options_test.go

```diff
@@ -237,3 +237,29 @@ func TestEnvMap(t *testing.T) {
 	m = utils.GetAsEqualsMap(l)
 	assert.Equal(t, m["foo"], "bar")
 }
+
+func TestWithSeparator(t *testing.T) {
+	t.Run("With default separator", func(t *testing.T) {
+		opts, err := NewProjectOptions([]string{
+			"testdata/simple/compose-with-network-and-volume.yaml",
+		}, WithName("my-project"))
+		assert.NilError(t, err)
+		p, err := ProjectFromOptions(opts)
+		assert.NilError(t, err)
+		assert.Equal(t, p.Networks["test-network"].Name, "my-project-test-network")
+
+	})
+
+	t.Run("With compatibility separator", func(t *testing.T) {
+		os.Setenv("COMPOSE_COMPATIBILITY", "true") //nolint:errcheck
+		defer os.Unsetenv("COMPOSE_COMPATIBILITY") //nolint:errcheck
```

Comment:
```suggestion
		t.Setenv("COMPOSE_COMPATIBILITY", "true")
```
It'll auto unset for you and ensure that the test doesn't get run in parallel by accident 🙂

Reply:
oooh this is nice to know!

Reply:
Good catch @milas, I always forgot about that one!!!